src/WpfMath/TexFormula.cs

```diff
@@ -49,18 +49,23 @@ public TexRenderer GetRenderer(TexStyle style, double scale, string systemTextFo
             return new TexRenderer(CreateBox(environment), scale);
         }
 
-        public void Add(TexFormula formula)
+        public void Add(TexFormula formula, SourceSpan source = null)

```

Comment:
The binary compatibility has been broken here, unfortunately (the source compatibility was preserved). We could fix that. Should we?

Reply:
Don't think so. Hopefully nobody uses that API. And we're still in API-breaking period of our semver (`0.x` versions).